### ISSUE TITLE
Routing with a Forwarding Address is missing

### ISSUE THREAD

**Issue by loreggiap** [MicrosoftDocs/OfficeDocs-Exchange#799]
If a Forwarding Address is specified on an Exchange Online mailbox then such forwarded mails are not routed through Centralized Mail Routing, even if this has been defined on the OutboundConnector.

Although the OutboundConnector has these parameters:
RouteAllMessagesViaOnPremises : True 
RecipientDomains : {*} 
UseMXRecord : False 
SmartHosts : {edgesvr}
EOP sends such mails directly to the Internet and not via the specified SmartHosts. 

This should be noted on this article, as it is definitely NOT what has been defined and what is being expected. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b002f0ce-5397-15cb-4c60-0eaaa87aaf52
* Version Independent ID: 32e3a594-2d2c-9f8a-9a38-e60e3ffaae85
* Content: [Transport routing in Exchange hybrid deployments](https://docs.microsoft.com/en-us/exchange/transport-routing#feedback)
* Content Source: [Exchange/ExchangeHybrid/transport-routing.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeHybrid/transport-routing.md)
* Product: **exchange-server-it-pro**
* Technology: ****
* GitHub Login: @dstrome
* Microsoft Alias: **dstrome**

**Comment by scanum** [comment]
@officedocsbot assign @scanum

**Comment by j0rt3g4** [comment]
For this, you'd need to make sure that the current domain is configured as Relay
![image](https://user-images.githubusercontent.com/5489996/58663001-13a7bd00-82f1-11e9-8ab4-781c05b4d422.png)

This is not a documentation issue. it's a technical issue.

**Comment by loreggiap** [comment]
It was worth a try, but changing the AcceptedDomain to DomainType = InternalRelay did not help.
I changed the parameter last week and did some tests this week, but the result remains the same: The settings of the OutboundConnectior are ignored.
The MessageTrace shows that the mail is relayed to the server defined by the MX-Record, although the OutboundConnector has UseMXRecord = False.

What is interesting is, that if the original mail comes from an Exchange Online mailbox then Centralized Routing is honored.
Setup:
  Exchange 2016 CU10 Mailbox and Edge Servers with Hybrid Setup
  OnPremMailbox1: onprem1@contoso.com
  OnPremMailbox2: onprem2@contoso.com
  O365Mailbox1: o365mbx1@contoso.com
  O365Mailbox2: o365mbx2@contoso.com - ForwardingAddress: onprem2@contoso.com
Mail 1:
  Mail is sent from: o365mbx1@contoso.com to: o365mbx2@contoso.com
  Because of the ForwardingAddress and the settings of the OutboundConnector the mail is routed through the Edge Server and arrives at OnPremMailbox2 (onprem2@contoso.com)
  The route is: o365mbx1@contoso.com -> o365mbx2@contoso.com -> ForwardingAddress -> Edge -> Exchange OnPrem -> onprem2@contoso.com
  This is what is expected.
Mail 2:
  Mail is sent from: onprem1@contoso.com to: o365mbx2@contoso.com
  The mail should take the same route: onprem1@contoso.com -> Edge -> Exchange Online -> o365mbx2@contoso.com -> ForwardingAddress -> Edge -> Exchange OnPrem -> onprem2@contoso.com
  That doesn't happen, but instead the route is: onprem1@contoso.com -> Edge -> Exchange Online -> o365mbx2@contoso.com -> ForwardingAddress -> MX-Record -> Internet Mail Server
  This is NOT expected and should be noted in the article or corrected in the code.

The routing is as follows:
![mail-routing-forwardingaddress2](https://user-images.githubusercontent.com/35289400/58795287-90d97780-85fa-11e9-95f8-3c4d9c40697c.png)

**Comment by scanum** [comment]
@loreggiap thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by scanum** [comment]
@officedocsbot close

**Comment by loreggiap** [comment]
Hello

Before posting a comment here, I had opened a service ticket. It was ticket # ‎13458305 and was open from ‎3‎/‎11‎/‎2019‎ until ‎34/‎115/‎2019‎‎:
https://portal.office.com/adminportal/home#/support/request?requestId=bfc3e50b-c78a-4798-afa5-76fdb5a8c767

I was directed to this article for the explanation: https://docs.microsoft.com/en-us/exchange/transport-routing#outbound-messages-to-the-internet and was asked to enter a DCR using https://office365.uservoice.com.
As indicated on my comments, I feel that the routing does not cover the situation with a ForwardingAddress involved. That's why I think that the article should cover this as well.

Kind regards,
Pierangelo

**Comment by j0rt3g4** [comment]
@loreggiap I've seen the ticket.
Thank you for being a premiere Customer, I work on "concierge and enterprise support" for O365 as well. I've contacted my support lead and see if he can go thru the case and help you to resolve it, the problem, here on GitHub, it's that we're not a "technical team" we document issues if something is wrong, so the support as @scanum  said, it's the best way to go.

About your case, I've contacted my SMEs on enterprise support, and talk about him about this issue, so if you allow us, he could probably give you a call and help you with the matter (I will be out today and tomorrow). The problem is that we should see all you got which usually takes time, but I'd go with the internal relay on the domains (internally and on Exchange online) and see if the centralized works.

Btw the centralized is over your on-premises (so all flow goes out and in from there) or cloud?
and the forwarding is for a 3rd party application?

**Comment by loreggiap** [comment]
Thank you for your reply and for your offer to help and find a solution to this problem.
As I already mentioned, we have already opened a case (Ticket # ‎13458305‎) and had to close it without having a solution or a workaround.
And I also mentioned that the suggestion to set the domains (on Premise and on Exchange Online) to "InternalRelay" did not work.

By the way, my company is not the only one having this problem. While entering a DCR I found this:
https://office365.uservoice.com/forums/273493-office-365-admin/suggestions/35162959-allow-forwarded-emails-in-centralised-transport-hy

For regulatory reasons we have to have Centralized Mail Transport working.
This process is described here: https://docs.microsoft.com/en-us/exchange/transport-routing, paragraph
"Route Internet-bound messages from Exchange Online through your on-premises organization (Centralized mail transport enabled)"
In most cases this works, as defined in the OutboundConnector, except when a ForwardingAddress is involved.
And we don't have 3rd party applications, it's only a ForwardingAddress, simple as that.

Kind regards,
Pierangelo
